src/Converter/ConverterInterface.php

```diff
@@ -17,4 +17,18 @@ public function convert(ElementInterface $element);
      * @return string[]
      */
     public function getSupportedTags();
+
+    /**
+     * @param \League\HTMLToMarkdown\ElementInterface $element
+     *
+     * @return void
+     */
+    public function openElement(ElementInterface $element);
+
+    /**
+     * @param \League\HTMLToMarkdown\ElementInterface $element
+     *
+     * @return void
+     */
+    public function closeElement(ElementInterface $element);
```

Comment:
Going with an event dispatcher is indeed one solution against major version bumping. However, it makes it less clear that converter might have that responsibility. At the same time, I do think that those two methods may not be appropriate for the ConverterInterface. I'm open to suggestions here.